### DISCUSSION TITLE
content position issue using `headerLargeTitle` from react navigation native stack

### DISCUSSION THREAD

**Discussion by crisfcodes** [react-navigation/react-navigation#10887] [Q&A]
I want to use [headerLargeTitle](https://reactnavigation.org/docs/native-stack-navigator/#headerlargetitle) feature from **react-navigation/native-stack**, but I'm having some issues with the content position.

## The problem: The content is not centered

Here are some examples with `headerTransparent: true` and with the default white color, as you can notice part of the content container is behind the header (the light salmon-colored box)

<div>
  <img src="https://i.stack.imgur.com/YPPY2.png" width="260" />
  <img src="https://i.stack.imgur.com/YOFy9.png" width="260" />
</div>

### Wrap the content with a scrollable view

According to the [docs](https://reactnavigation.org/docs/native-stack-navigator/#headerlargetitle) you can wrap the content with a `scrollView` or `flatList` with the `contentInsetAdjustmentBehavior="automatic"` prop defined, what this does is to move the content by adding a content insets, this brings other problems since it makes the content scrollable and doesn't solve the centering problem either.

[![enter image description here][1]][1]

### Use an extra `<SafeAreaView />`

Wrap my content in a `<SafeAreaView />` and put inside an aditional `<SafeAreaView />`, this seems to solve the problem on the **iPhon13** but on smaller devices like the **iPhone8**(where the size of the safe area changes), this doesn't work. here is an example code:

```typescript
<SafeAreaVie>
  <SafeAreaView />
  <View>
    <Text>Some Text</Text>
  </View>
</SafeAreaView>
```

### What I want

I want to use `headerLargeTitle` feature and the content to start just below the header without wrapping the content with `ScrollView` with `contentInsetAdjustmentBehavior="automatic"`.  

**NOTE**: here is a [snack](https://snack.expo.dev/@cris0501/headerlargetitle-issue) with the example code used in the question

Any suggestion: @satya164 

  [1]: https://i.stack.imgur.com/Xa40O.gif

**Comment by timothyerwin** [comment]
did you find any solution to this? the scrollview works with contentInsetAdjustmentBehavior="automatic" but flatlist does not set the insets correctly.

**Reply by timothyerwin** [reply]
actually, it sets correctly unless it's inverted. I was able to solve it with a list footer component with an empty view with inset top height
